onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -1155,7 +1191,122 @@ Status BitShift<T>::Compute(OpKernelContext* context) const {
 }
 
 template <typename T>
-class Sin final : public OpKernel {
+Status BitwiseAnd<T>::Compute(OpKernelContext* context) const {
```

Comment:
It looks like a lot of boilerplate code that should be common to all binary elementwise ops. Just curious why there is no common template for this.